src/components/form/form.mdx

```diff
@@ -0,0 +1,17 @@
+---
+name: Form
+route: /form
+menu: Components
+---
+
+# Form
+
```

Comment:
I assume these docs will be visible once we create a new release, even if the component itself isn't bundled into the release, so we should write something about the component not being available yet.

Reply:
I was hoping it would not be visible, but you might be right! The `excludeSrc` property will probably not work on the docs files. Maybe there is some other way of hiding it :thinking: